dev/Playground.vue

```diff
@@ -1,49 +1,14 @@
 <template>
-    <div>
```

Comment:
want me to add a cleanup script to the precommit?

This will wipe the `Playground.vue` file

Reply:
i don't think I would want it wiped on every commit, that might get annoying